Sources/Credentials/MultiTypeSafeCredentials.swift

```diff
@@ -48,13 +48,13 @@ import Foundation
  */
 public protocol TypeSafeMultiCredentials: TypeSafeCredentials {
     
-    /// An array of authentication types that conform to `TypeSafeCredentials`. The `authenticate` function for each type will be called in order and, on successfully authenticating, will call `init` using the `TypeSafeCredentials` instance.
+    /// An array of authentication types that conform to `TypeSafeCredentials`. The `authenticate` function for each type will be called in order. If a type successfully authenticates, it will call `init` using the instance of themselves.
```

Comment:
I'd prefer:
> An array of authentication types that conform to `TypeSafeCredentials`. Each type will be asked, in turn, to authenticate an incoming request. If a type successfully authenticates, it will call `init(successfulAuth:)` with an instance of itself.